pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -185,6 +186,7 @@ class LightningKokkos(LightningBase):
         operations = allowed_operations
         observables = allowed_observables
         _backend_info = backend_info
+        config = Path(__file__).parents[1] / "core" / "src" / "lightning_kokkos.toml"
```

Comment:
Wow, is division overloaded with path concatenation? 😅

Reply:
This was also new to me. Erick shared the link to [docs](https://github.com/python/cpython/blob/3.12/Lib/pathlib.py#L719-L723).